libtalek/handle_test.go

```diff
@@ -55,8 +55,8 @@ func TestSerialization(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not deserialize: %v\n", err)
 	}
-	if !bytes.Equal(h.SharedSecret[:], h2.SharedSecret[:]) {
-		t.Fatalf("serialization log info!")
+	if !bytes.Equal(h.SharedSecret[:], h2.SharedSecret[:]) || !bytes.Equal(h.SigningPublicKey[:], h2.SigningPublicKey[:]) {
```

Comment:
seeds have tests. seqNo is 0, in these tests so doesn't matter. The assumption is just that sscanf fills, i'm not convinced we gain much more confidence with the extra verbosity